ppl/archive/chunk/writer.go

```diff
@@ -31,30 +34,59 @@ type Writer struct {
 	wroteFirst     bool
 }
 
-func NewWriter(ctx context.Context, dir iosrc.URI, order zbuf.Order, masks []ksuid.KSUID, opts zngio.WriterOpts) (*Writer, error) {
+type Options struct {
+	Order       zbuf.Order
+	Masks       []ksuid.KSUID
+	Definitions index.Definitions
+	ZngWriter   zngio.WriterOpts
```

Comment:
Maybe sort? And maybe call ZngWriter just Zng for consistency with zio.WriterOpts.Zng?